<!DOCTYPE html>
<html>
<head>
  <meta name="google-site-verification" content="W5tAKU2rc92r8h9yQ98AohwSaYlAe66HZJAyJreCvIk" />
  <title>BioInfoHub</title>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <link rel="stylesheet" href="styles.css">
</head>
<body>
  <header>
    <div class="header-container">
      <div class="logo-container">
        <img src="logo.png" alt="BioInfoHub Logo" class="header-logo">
      </div>
      <div class="title-container">
        <h1>BioInfoHub</h1>
        <span class="logo-slogan">Connecting Bioinformaticians, Empowering Discoveries!</span>
      </div>
    </div>
    <nav>
      <ul>
        <li><a href="#" onclick="showSection('home')" class="active">Home</a></li>
        <li><a href="#" onclick="showSection('forum')">Forum</a></li>
        <li><a href="#" onclick="showSection('tools')">Tools</a></li>
        <li><a href="#" onclick="showSection('news')">News</a></li> 
        <li><a href="#" onclick="showSection('blog')">Blogs</a></li> 
        <li><a href="#" onclick="showSection('project')">Projects</a></li>
        <li><a href="#" onclick="showSection('contact')"class="fab">Contact Us</a></li>
      </ul>
    </nav>
  </header>
  
  <main>
    <section id="home">
      <div class="home-container">
        <h1>Welcome to BioInfoHub</h1>
        <div class="about-container">
          <h2>About BioInfoHub</h2>
          <p>BioInfoHub is a vibrant online community dedicated to bringing together bioinformaticians from around the globe. Our platform serves as a central hub for professionals, researchers, and enthusiasts in the field of bioinformatics, fostering collaboration, knowledge exchange, and empowerment.</p>
          <p>Join us to connect with a diverse network of bioinformaticians, spanning various domains and expertise. Whether you're a seasoned bioinformatician or just starting your journey in this exciting field, BioInfoHub provides a supportive environment to grow, learn, and share experiences.</p>
        </div>
        
        <div class="offer-container">
          <h2>What We Offer</h2>
          <ul>
            <li><strong>Connect with Experts:</strong> Engage with leading bioinformaticians and industry professionals through forums, discussion boards, and networking events. Share your ideas, seek advice, and build valuable connections within the community.</li>
            <li><strong>Stay Informed:</strong> Access the latest news, trends, and breakthroughs in bioinformatics. Our curated content keeps you updated on cutting-edge research, emerging technologies, and industry developments, ensuring you stay at the forefront of the field.</li>
            <li><strong>Discover Resources:</strong> Explore a vast array of resources, including tutorials, datasets, software tools, and educational materials. Whether you're looking to enhance your skills or dive into a new area of bioinformatics, BioInfoHub is your go-to source.</li>
            <li><strong>Collaborate on Projects:</strong> Find potential collaborators for your research projects and contribute to ongoing initiatives. BioInfoHub provides a platform for interdisciplinary collaborations, enabling you to leverage collective expertise and accelerate discoveries.</li>
            <li><strong>Career Development:</strong> Boost your professional growth with career resources, job postings, and mentorship opportunities. Connect with mentors who can guide you on your bioinformatics career path and access exclusive industry insights.</li>
          </ul>
        </div>
      </div>
    </section>
    <section id="forum">
      <div class="forum-container">
        <h1>Join the BioInfoHub Forum</h1>
        <p>Our forum is a vibrant space for bioinformaticians to discuss ideas, share knowledge, and seek assistance. Engage with fellow community members, ask questions, and contribute your expertise to help others.</p>
        <a href="https://t.me/+IDTY7CjbskBmZTg9" class="forum-button">Visit the Forum</a>
      </div>
    </section>
    <section id="news">
      <div id="news-container"></div>
    </section>
    <section id="projectsSection">
    </section>
<section id="project">
  <div class="projects-container">
    <h2 class="section-heading">Projects</h2>
    <div class="section-content">
      <p>We are currently working on exciting projects! Stay tuned for updates.</p>
      <p>We will notify you soon with more information about our projects and how you can get involved.</p>
    </div>
  </div>
</section>
<section id="tools">
      <div class="tools-container">
        <h2 class="section-heading">Bioinformatics Tools</h2>
        <div class="section-content">
          <h3>Genomics Tools</h3>
          <ul>
            <li><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" target="_blank">BLAST</a> - (Basic Local Alignment Search Tool) A tool for sequence similarity searches against a database of nucleotide or protein sequences.</li>
             <li><a href="http://bowtie-bio.sourceforge.net/index.shtml" target="_blank">Bowtie</a> - A fast and memory-efficient tool for aligning sequencing reads to reference genomes.</li>
             <li><a href="http://www.htslib.org/" target="_blank">SAMtools</a> - A suite of utilities for manipulating and analyzing SAM/BAM format alignment files.</li>
            <li><a href="https://genome.ucsc.edu/" target="_blank">Genome Browser</a> - A tool for visualizing and exploring genomic data, including gene annotations, genetic variants, and other genomic features. </li> 
            <li><a href="https://galaxyproject.org/" target="_blank">Galaxy</a> -  An open-source, web-based platform for data-intensive biomedical research, including genomics analysis workflows. </li>
            <li><a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">FastQC</a> -  A quality control tool for high-throughput sequence data, providing detailed reports on sequence quality metrics. </li>
            <li><a href="https://github.com/trinityrnaseq/trinityrnaseq/wiki" target="_blank">Trinity</a> -  A software package for de novo reconstruction of transcriptomes from RNA-seq data. </li>
            <li><a href="https://software.broadinstitute.org/software/igv/" target="_blank">IGV (Integrative Genomics Viwer)</a> - A high-performance visualization tool for interactive exploration of large genomic datasets.</li>
            <li><a href="https://broadinstitute.github.io/picard/" target="_blank">Picard</a> - Picard is a Java-based toolkit for manipulating and analyzing next-generation sequencing data files (SAM/BAM/CRAM). It offers a wide range of utilities for processing and quality control of sequencing data.</li>
           <li><a href="https://www.ensembl.org/" target="_blank">Ensembl</a> - Ensembl is a genome browser and annotation database that provides access to various genomes, gene annotations, and functional genomics data. It allows you to explore gene structures, variations, and regulatory elements.</li>
          </ul>
          
          <h3>Transcriptomics Tools</h3>
          <ul>
            <li><a href="https://bioconductor.org/packages/release/bioc/html/DESeq2.html" target="_blank">DESeq2</a> - A widely used R/Bioconductor package for differential gene expression analysis from RNA-seq data.</li>
            <li><a href="http://cole-trapnell-lab.github.io/cufflinks/" target="_blank">Cufflinks</a> - A suite of tools for transcriptome assembly, differential gene expression, and transcript abundance estimation from RNA-Seq data.</li>
           <li><a href="https://bioconductor.org/packages/release/bioc/html/edgeR.html" target="_blank">edgeR</a> - A bioconductor package for differential expression analysis of RNA-seq data, using empirical Bayes estimation.</li>
           <li><a href="https://salmon.readthedocs.io/" target="_blank">Salmon</a> - A tool for quantifying transcript abundance from RNA-seq data, providing fast and accurate transcript-level quantification.</li>
           <li><a href="https://ccb.jhu.edu/software/stringtie/" target="_blank">StringTie</a> - A software package for transcript assembly and quantification of RNA-seq data.</li>
           <li><a href="https://www.bioconductor.org/packages/release/bioc/html/ballgown.html" target="_blank">Ballgown</a> - A software package for differential expression analysis and visualization of RNA-seq data.</li>
           <li><a href="https://github.com/alexdobin/STAR" target="_blank">STAR</a> - A fast and accurate RNA-seq aligner, suitable for both transcriptome and whole-genome alignment.</li>
           <li><a href="http://www.usadellab.org/cms/?page=trimmomatic" target="_blank">Trimmomatic</a> - A tool for trimming and filtering sequencing reads based on quality scores. It helps remove low-quality bases, adapter sequences, and other artifacts. </li>
          </ul>
          
          
          <h3>Metabolomics Tools</h3>
          <ul>
            <li><a href="https://cytoscape.org/" target="_blank">Cytoscape</a> - Although not specific to metabolomics, Cytoscape is a powerful software platform widely used for network visualization and analysis. It can be employed to integrate and analyze metabolomics data in the context of biological networks, such as metabolic pathways or protein-protein interactions. Cytoscape supports various plugins that provide additional functionalities for metabolomics data analysis.</li>
            <li><a href="https://www.msea.ca/" target="_blank">MSEA (Metabolite Set Enrichment Analysis)</a> - MSEA is a web-based tool for pathway analysis and metabolite set enrichment analysis. It helps in understanding the biological context of metabolomics data by identifying overrepresented metabolic pathways and functional modules.</li>
              <li><a href="http://mzmine.github.io/" target="_blank">MZmine</a> - An open-source software for processing and visualization of mass spectrometry-based metabolomics data. It offers a variety of modules for peak detection, alignment, quantitation, and identification.</li>
           </ul>
          
          <h3>Proteomics Tools</h3>
          <ul>
            <li><a href="https://www.rcsb.org/" target="_blank">Protein Data Bank (PDB)</a> - An archive of experimentally determined three-dimensional protein structures, allowing for structure visualization and analysis.</li>
            <li><a href="https://www.uniprot.org/" target="_blank">UniProt</a> - UniProt is a comprehensive protein sequence and functional information resource. It provides access to a large collection of protein sequences and annotations.</li>
            <li><a href="https://www.expasy.org/" target="_blank">Expasy Proteomics Tools</a> - Expasy is a collection of bioinformatics tools for proteomics data analysis. It includes tools for protein identification, characterization, post-translational modifications, and more.</li>
            <li><a href="https://www.expasy.org/spdbv/" target="_blank">Swiss-PdbViewer (DeepView)</a> - A molecular visualization and analysis tool for protein structures. It allows interactive manipulation, analysis, and visualization of protein structures.</li>
            <li><a href="https://pymol.org/" target="_blank">PyMOL</a> - A powerful molecular graphics system that enables visualization, analysis, and rendering of protein structures. It offers a wide range of features and scripting capabilities.</li>
            <li><a href="https://string-db.org/" target="_blank">STRING</a> -  A database and web resource that provides information about protein-protein interactions, functional associations, and networks. It integrates data from various sources and offers visualization tools.</li>
           <li><a href="https://david.ncifcrf.gov/" target="_blank">DAVID</a> -  DAVID (Database for Annotation, Visualization, and Integrated Discovery) is a web-based tool for functional annotation and enrichment analysis of proteins. It helps in identifying enriched biological themes, functional annotations, and pathway analysis for a given set of proteins.</li>
   
          </ul>
          <h3>Computer Aided Drug Designing & Structural Bioinformatics</h3>
          <ul>
            <li><a href="http://autodock.scripps.edu/resources/adt" target="_blank">AutoDock Tools</a> - AutoDock Tools is a graphical user interface (GUI) used for preparing and analyzing molecules for molecular docking simulations. It is commonly used in conjunction with AutoDock, a popular molecular docking software. AutoDock Tools allows users to visualize and manipulate ligands and receptors, set up docking parameters, and analyze the results.</li>
            <li><a href="http://vina.scripps.edu/" target="_blank">AutoDock Vina</a> - AutoDock Vina is a molecular docking program that is based on the AutoDock algorithm. It is used for predicting the binding affinity and orientation of small molecules (ligands) to a target protein (receptor). AutoDock Vina provides a command-line interface and is known for its speed and accuracy.</li>
            <li><a href="http://pyrx.sourceforge.net/" target="_blank">PyRx</a> - PyRx (Python Prescription) is a virtual screening software that integrates various open-source tools and provides a user-friendly interface for molecular docking studies. It uses AutoDock Vina as the default docking engine and allows users to prepare, dock, and analyze small molecules against target proteins.</li>
            <li><a href="http://www.gromacs.org/" target="_blank">GROMACS</a> - GROMACS (Groningen Machine for Chemical Simulations) is a versatile molecular dynamics simulation package widely used in computational chemistry and biophysics. It allows researchers to study the motion and behavior of biomolecules, such as proteins and lipids, at the atomic level. GROMACS provides powerful simulation algorithms and analysis tools.</li>
            <li><a href="https://www.swissadme.ch/">SwissADME</a> - A powerful molecular graphics system that enables visualization, analysis, and rendering of protein structures. It offers a wide range of features and scripting capabilities.</li>
            <li><a href="https://plasma-gate.weizmann.ac.il/Grace/" target="_blank">Grace</a> - Grace (Graphing, Advanced Computation, and Exploration) is a plotting tool used for data visualization and analysis. It provides a wide range of features to create publication-quality graphs, including 2D and 3D plots, mathematical functions, curve fitting, and data manipulation. Grace is often used in scientific research and data analysis.</li>
           <li><a href="http://openbabel.org/" target="_blank">Open Babel</a> -  Open Babel is an open-source chemical toolbox designed to speak the many languages of chemical data. It allows users to convert, analyze, and manipulate chemical file formats. Open Babel supports a wide range of chemical file formats and provides a command-line interface as well as a programming API.</li>
          <li><a href="https://www.rbvi.ucsf.edu/chimerax/" target="_blank">ChimeraX</a> -  ChimeraX is a next-generation molecular visualization program developed by the UCSF Chimera team. It allows users to visualize and analyze molecular structures and complexes in 3D. ChimeraX supports a wide range of molecular file formats and provides advanced visualization features, including interactive model manipulation and virtual reality support.</li>
          </ul>
          
          <h3>Programming Tools</h3>
          <ul>
            <li><a href="https://www.r-project.org/" target="_blank">R</a> -  R is a powerful statistical programming language widely used in bioinformatics. It provides numerous packages and libraries for data analysis, visualization, and statistical modeling.</li>
           <li><a href="https://www.bioconductor.org/" target="_blank">Bioconductor</a> -  R is a powerful statistical programming language widely used in bioinformatics. It provides numerous packages and libraries for data analysis, visualization, and statistical modeling.</li>
          <li><a href="https://biopython.org/" target="_blank">Biopython</a> -  Biopython is a collection of freely available Python tools for computational biology and bioinformatics. It provides modules for sequence manipulation, protein analysis, file parsing, and much more.</li>
          <li><a href="https://bioperl.org/" target="_blank">BioPerl</a> -  BioPerl is a collection of Perl modules and libraries for bioinformatics tasks. It offers a comprehensive set of modules for sequence analysis, file parsing, database access, and other bioinformatics-related tasks.</li>
           
          </ul>
        </div>
      </div>
    </section>
<section id="blog">
    <div class="blog-container">
        <h2 class="section-heading">Stay informed with our latest blog posts</h2>
        <div class="section-content">
            <div class="blog-post">
                <img src="Bioinformatics.png" alt="Bioinformatics Image" class="blog-image">
                <div class="blog-description">
                    <h3>What is Bioinformatics?</h3>
                    <p><strong>Written by Piyush Kumar</strong></p>
                    <p><strong>Date: July 10, 2023</strong></p>
                    <p>Bioinformatics is a field that combines biology and computer science to study and analyze biological data. It involves various areas such as Sequence Analysis, Genomic Analysis, Structural Bioinformatics, Systems Biology, Data Mining, and Machine Learning.</p>
                    <div class="blog-details hidden">
                        <ul>
                            <li>Sequence Analysis: One of the fundamental aspects of bioinformatics is sequence analysis. This involves the study of DNA, RNA, and protein sequences to understand their structure, function, and relationships. Sequence alignment methods are used to compare sequences and identify similarities, which can provide insights into evolutionary relationships, identify conserved regions, and predict functional elements.</li>
                            <li>Genomic Analysis: Genomic analysis focuses on studying entire genomes, including the organization, structure, and function of genes within an organism's DNA. Bioinformatics tools are used to annotate genomes, identify genes, predict their functions, and analyze variations within and between genomes. It also involves comparative genomics, which compares genomes of different species to understand evolutionary relationships and identify genomic variations associated with traits and diseases.</li>
                            <li>Structural Bioinformatics: Structural bioinformatics deals with the prediction and analysis of three-dimensional structures of biomolecules, primarily proteins. Computational methods are used to predict protein structures based on amino acid sequences and to understand their functions, interactions, and dynamics. This information is crucial for drug discovery, protein engineering, and understanding the mechanisms of biological processes.</li>
                            <li>Systems Biology: Systems biology is an interdisciplinary approach that combines experimental data with computational modeling to understand complex biological systems as a whole. Bioinformatics plays a key role in integrating and analyzing various types of high-throughput data, such as genomics, transcriptomics, proteomics, and metabolomics, to build models and gain insights into how different components of a biological system interact and function.</li>
                            <li>Data Mining and Machine Learning: With the ever-increasing amount of biological data generated, bioinformatics relies on data mining and machine learning techniques to extract meaningful patterns and knowledge from large datasets. These methods help in identifying biomarkers, predicting protein functions, classifying biological samples, and discovering novel relationships or associations within biological data. Data mining and machine learning approaches are essential for handling the complexity and scale of biological information and enabling data-driven discoveries.</li>
                        </ul>
                    </div>
                    <button class="read-more-btn" onclick="toggleBlogDetails(this)">Read More</button>
                </div>
            </div>
            <div class="blog-post">
                <img src="manhattan.png" alt="Bioinformatics Image" class="blog-image">
                <div class="blog-description">
                    <h3>Manhattan Distance in Gene Expression Analysis</h3>
                    <p><strong>Written by Ankur Singh</strong></p>
                    <p><strong>Date: August 11, 2023</strong></p>
                    <p>Manhattan Distance is a way to measure the "distance" between two points in a grid-like space, just like navigating through the streets of Manhattan. It's also known as L1 distance or taxicab distance.</p>
                    <div class="blog-details hidden">
                        <p>Manhattan Distance is widely used in bioinformatics to analyze and compare biological data, especially in genomics and gene expression studies. One common application is in the analysis of microarray data.</p>
                        <h2>Example: Comparing Gene Expression Patterns</h2>
                        <p>Imagine we have a dataset containing the expression levels of three genes (A, B, and C) in four different tissue samples (Tissue 1, Tissue 2, Tissue 3, and Tissue 4). The gene expression levels are measured as numerical values.</p>
                        <table border="1">
                            <tr>
                                <th>Gene</th>
                                <th>Tissue 1</th>
                                <th>Tissue 2</th>
                                <th>Tissue 3</th>
                                <th>Tissue 4</th>
                            </tr>
                            <tr>
                                <td>A</td>
                                <td>5</td>
                                <td>3</td>
                                <td>9</td>
                                <td>7</td>
                            </tr>
                            <tr>
                                <td>B</td>
                                <td>2</td>
                                <td>6</td>
                                <td>4</td>
                                <td>8</td>
                            </tr>
                            <tr>
                                <td>C</td>
                                <td>1</td>
                                <td>5</td>
                                <td>3</td>
                                <td>7</td>
                            </tr>
                        </table>
                        <h3>Calculating Manhattan Distance:</h3>
                        <p>For gene A: Manhattan Distance = |3 - 5| + |5 - 3| = 2 + 2 = 4</p>
                        <p>For gene B: Manhattan Distance = |6 - 2| + |5 - 3| = 4 + 2 = 6</p>
                        <p>For gene C: Manhattan Distance = |5 - 1| + |5 - 3| = 4 + 2 = 6</p>
                        <h3>Interpretation:</h3>
                        <p>The Manhattan Distance values for each gene between Tissue 1 and Tissue 2 are as follows:</p>
                        <ul>
                            <li>Gene A: 4</li>
                            <li>Gene B: 6</li>
                            <li>Gene C: 6</li>
                        </ul>
                        <p>Lower Manhattan Distance values indicate a higher similarity in gene expression patterns. In this example, we can see that Gene A has the lowest Manhattan Distance of 4, indicating that its expression pattern is more similar between Tissue 1 and Tissue 2 compared to the other two genes. On the other hand, Gene B and Gene C both have a Manhattan Distance of 6, suggesting that their expression patterns are less similar between the two tissue samples.</p>
                        <p>By calculating the Manhattan Distance for each gene, we gained insights into the similarities and differences in gene expression, helping us understand the underlying biology and potential relationships between the tissues.</p>
                      <p>For more detailed information, you can download the document here: <a href="https://t.me/c/1921990809/1/99" target="_blank">Download Document</a>.</p>

                    </div>
                    <button class="read-more-btn" onclick="toggleBlogDetails(this)">Read More</button>
                </div>
            </div>
            <div class="blog-post">
                <img src="wgcna.png" alt="Bioinformatics Image" class="blog-image">
                <div class="blog-description">
                    <h3>Weighted Gene Co-Expression Network Analysis (WGCNA)</h3>
                    <p><strong>Written by Piyush Kumar</strong></p>
                    <p><strong>Date: August 11, 2023</strong></p>
                    <p>Weighted Gene Co-Expression Network Analysis (WGCNA) is a powerful bioinformatics method used to analyze gene expression data. It identifies co-expression patterns between genes and groups genes into modules based on their similar expression profiles. WGCNA is widely used to understand complex regulatory mechanisms within cells, especially in diseases like cancer.</p>
                    <div class="blog-details hidden">
                        <ul>
                            <li><strong>Identifying Co-Expressed Gene Modules:</strong> WGCNA identifies groups of genes (modules) with similar expression patterns.</li>
                            <li><strong>Dimensionality Reduction:</strong> WGCNA reduces high-dimensional data into manageable modules.</li>
                            <li><strong>Understanding Gene Regulatory Networks:</strong> WGCNA uncovers gene interactions and regulatory networks.</li>
                            <li><strong>Identifying Biomarkers:</strong> WGCNA identifies potential biomarkers for diseases.</li>
                            <li><strong>Relationship with Phenotypes:</strong> Correlates modules with traits, diseases, or conditions.</li>
                            <li><strong>Functional Enrichment Analysis:</strong> Identifies enriched pathways within modules.</li>
                            <li><strong>Reproducibility and Robustness:</strong> WGCNA is a validated and reproducible method.</li>
                            <li><strong>Integration of Multiple Data Types:</strong> WGCNA integrates different omics data.</li>
                            <li><strong>Systems Biology Insights:</strong> Provides a holistic view of complex interactions.</li>
                        </ul>
                        <p>Weighted Gene Co-expression Network Analysis (WGCNA) in the context of cancer research involves the following steps:</p>
                        <ul>
                            <li><strong>Data Preprocessing:</strong> Import, clean, normalize, and handle missing values in gene expression data.</li>
                            <li><strong>Co-expression Network Construction:</strong> Create a weighted correlation network and identify gene modules.</li>
                            <li><strong>Module Preservation and Eigengene Calculation:</strong> Validate module preservation and calculate module eigengenes.</li>
                            <li><strong>Module-Trait Relationship:</strong> Associate modules with clinical traits and analyze correlations.</li>
                            <li><strong>Hub Gene Identification:</strong> Identify intramodular hub genes based on connectivity.</li>
                            <li><strong>Functional Enrichment Analysis:</strong> Perform GO and pathway enrichment analysis.</li>
                        </ul>
                        <h3>Conclusion:</h3>
                        <p>WGCNA is a valuable approach in cancer research, enabling the identification of co-expression patterns and regulatory networks within gene expression data. It aids in understanding disease mechanisms, identifying biomarkers, and exploring potential therapeutic targets.</p>
                      <p>For more detailed information, you can download the document here: <a href="https://t.me/c/1921990809/1/100" target="_blank">Download Document</a>.</p>

                    </div>
                    <button class="read-more-btn" onclick="toggleBlogDetails(this)">Read More</button>
                </div>
            </div>
        </div>
    </div>
</section>
<section id="contact">
  <div class="contact-container">
    <h2>Contact Us</h2>
    <p>For inquiries and further information, please reach out to us:</p>
    <div class="contact-details">
      <p>Email: <a href="mailto:bioinfohub2023@gmail.com">bioinfohub2023@gmail.com</a></p>
      <p>Follow us on:</p>
      <div class="social-buttons">
        <a href="https://www.instagram.com/bioinfohub/" target="_blank" rel="noopener noreferrer">
          <img src="Instagram.png" alt="Instagram">
        </a>
        <a href="https://www.linkedin.com/in/bioinfohub-b06a5a27b" target="_blank" rel="noopener noreferrer">
          <img src="LinkedIn.png" alt="LinkedIn">
        </a>
      </div>
    </div>
  </div>
</section>


 <footer>
    <div class="footer-container">
      <div class="footer-social">
        <a href="https://www.linkedin.com/in/bioinfohub-b06a5a27b"><img src="LinkedIn.png" alt="LinkedIn"></a>
        <a href="https://www.instagram.com/bioinfohub/"><img src="Instagram.png" alt="Instagram"></a>
        <a href="mailto:bioinfohub2023@gmail.com"><img src="Gmail.png" alt="Gmail"></a>
      </div>
      <p>&copy; 2024 BioInfoHub. All rights reserved. | Site Owner: Ankur Singh       </p>
    </div>
  </footer>
  
  <script src="https://unpkg.com/rss-parser/dist/rss-parser.min.js"></script>
  <script src="script.js"></script>
</body>
</html>

